src/BuildScriptGeneratorCli/Commands/PrepareEnvironmentCommand.cs

```diff
@@ -0,0 +1,350 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using McMaster.Extensions.CommandLineUtils;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.Ini;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.Oryx.BuildScriptGenerator;
+using Microsoft.Oryx.BuildScriptGeneratorCli.Options;
+using Microsoft.Oryx.Common;
+
+namespace Microsoft.Oryx.BuildScriptGeneratorCli
+{
+    [Command(Name, Description = "Sets up environment by detecting and installing platforms.")]
+    internal class PrepareEnvironmentCommand : CommandBase
```

Comment:
The name maybe a little ambiguous, we could think of a clear name for this command?

Reply:
Since the command shortcut is called `prep` (which means prepare), I wanted to keep it in align with it.